vm/vmgs/vmgs/Cargo.toml

```diff
@@ -28,6 +29,7 @@ vmgs_format.workspace = true
 inspect = { workspace = true, optional = true }
 inspect_counters = { workspace = true, optional = true }
 mesh_protobuf = { workspace = true, optional = true }
+mesh = { workspace = true, optional = true }
```

Comment:
Note that we already have mesh_protobuf in some cases, and we only need Protobuf.